### ISSUE TITLE
Create GeoPackage importer

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#157]
## Description
See Epic: #156 

## Acceptance criteria
* Download geoPackage data
* Ingest geoPackage data into database

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dancingsushii** [comment]
Obsolete as the importer’s type was SHAPE format.
